recipe/meta.yaml

```diff
@@ -9,10 +9,10 @@ package:
 source:
   fn: {{ version }}.tar.gz
   url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
-  sha256: cadeefa9a824671bdcad790a075efbaa0d5a00458c43c9b4ce7110b129df7b10
+  sha256: 15ff8f24909c68340532147ca6ff68e05508c836ceb5307c23d80a825a3e2a45
 
 build:
-  number: 1
+  number: 0
```

Comment:
You need to add here,
```
 skip: true  # [py<35]
```
then re-rerender, as the minimal python versions is not parsed from `setup.py` I think.